samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -0,0 +1,676 @@
+.. _broadcast_configuration_tool:
+
+Broadcast Configuration Tool
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Broadcast Configuration Tool application implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>`.
+
+In this mode, transmitting broadcast audio happens using Broadcast Isochronous Stream (BIS) and Broadcast Isochronous Group (BIG).
```

Comment:
Should mention Auracast here as well.

Reply:
Nit 2.0. Depending on the config, it may or may not be an Auracaster.
Suggest:

The Broadcast Configuration Tool sample implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>` and may act as an Auracast broadcaster if an Auracast preset is used.